source/agg-pipeline-builder/export-pipeline-results.txt

```diff
@@ -0,0 +1,65 @@
+.. _export-agg-builder-results:
+
+===================================
+Export Aggregation Pipeline Results
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To use your aggregated data in other external tools, you can export your 
+pipeline results from |compass| as a JSON or CSV file. 
+
+About This Task 
+---------------
+
+Avoid exporting to CSV files when possible. CSV files may lose type information 
+and are not suitable for backing up your data.
```

Comment:
[suggestion]

I think rather than a blanket warning of basically "avoid CSVs all the time", it might be helpful to narrow in on their downsides. Even though you shouldn't use CSVs for backups, I think there are other use cases that we might be discounting with this copy.

Maybe we can tweak this to be something like: "Avoid using CSV exports to back up your data. CSV files may lose type information and are not suitable for backups and restores."

Reply:
Ah gotcha! Thanks for the suggestion. Updated!